src/dtls13.c

```diff
@@ -1453,17 +1464,24 @@ int Dtls13ParseUnifiedRecordLayer(WOLFSSL* ssl, const byte* input,
         hdrInfo->recordLength = inputSize - idx;
     }
 
-    /* minimum size for a dtls1.3 packet is 16 bytes (to have enough ciphertext
-       to create record number xor mask). (draft 43 - Sec 4.2.3) */
-    if (hdrInfo->recordLength < DTLS13_RN_MASK_SIZE)
-        return LENGTH_ERROR;
-    if (inputSize < idx + DTLS13_RN_MASK_SIZE)
-        return BUFFER_ERROR;
+#ifdef HAVE_NULL_CIPHER
```

Comment:
can we avoid macros around the if statement but not the body of the if?
maybe remove the  macros altogether or move the encryption at the end and exit if cipher is null, or move all check inside Dtls13EncryptDecryptRecordNumber